### ISSUE TITLE
Run on MacOS?

### ISSUE THREAD

**Issue by mobile26** [Kemmey/Kemmey-TeslaWatch-Public#248]
"as it runs on iPads, phones and **macs** as well". How do I do that, or is it only for M1?

**Comment by Kemmey** [comment]
Yeah, m1 for now - I'll let you know when I have a Mac beta to share that works on all macs. I have it running, but scrolling was super buggy, so I've held it back...

**Comment by mobile26** [comment]
Thank you

Here is the API for Steering Wheel Heater 

/api/1/vehicles/{id}/command/remote_steering_wheel_heater_request



Fra: Kim Hansen <notifications@github.com>
Svar: Kemmey/Kemmey-TeslaWatch-Public <reply@reply.github.com>
Dato: 23. desember 2020 at 13:56:25
Til: Kemmey/Kemmey-TeslaWatch-Public <kemmey-teslawatch-public@noreply.github.com>
Kopi: mobile26 <foras@fastmail.fm>, Author <author@noreply.github.com>
Emne:  Re: [Kemmey/Kemmey-TeslaWatch-Public] Run on MacOS? (#248)  

Yeah, m1 for now - I'll let you know when I have a Mac beta to share that works on all macs. I have it running, but scrolling was super buggy, so I've held it back...

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.

_____________________
Gunnar Altberg-Andresen
   Mobile:+47 970 10 970
